src/Core/Silk.NET.Core/Silk.NET.Core.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFrameworks>netstandard2.0;netcoreapp3.1;net5.0</TargetFrameworks>
+        <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0</TargetFrameworks>
```

Comment:
Not sure what's going on here, I don't think we properly handle netstandard2.1 when loading libraries yet.

Reply:
I don't think we have any TFM-specific loading code, we generally use the deps.json if available.